### ISSUE TITLE
Dimensions of s,u,v and Ztilde

### ISSUE THREAD

**Issue by reyhashemi** [federhub/ST_DeepLearning#2]
Hi,
Thank you for sharing your code and the interesting job.
It would be grateful if you could please print the shape of `s, u, v`, and Ztilde in `s, u, v = tf.linalg.svd(Ztilde)`?

`

**Comment by Alezavala97** [comment]
Hi,
I carried out a similar project based on what Amato proposed and one of the key aspects of my work is that I used openly available data. This means you can freely download and run the data on your own so you can replicate it with your own data.

If you have questions or suggestions for improvement, contact me since I am just working with this methodology.

[](https://github.com/Alezavala97/ST_DeepLearning-implemented-in-Chile/blob/main/C%C3%B3digos/30-70%20zona%20central%20SVD-DL.ipynb)

**Comment by reyhashemi** [comment]
@Alezavala97 Yes, sure, Thanks! I have actually several points to discuss, in particular regarding the network and its training.
how could I contact you? I did not find any email addresses on your page.

**Comment by Alezavala97** [comment]
@reyhashemi You can write to my email alejandro.zavala.zavalayg@gmail.com greetings!
